app/src/main/java/edu/drexel/lapcounter/lapcounter/frontend/DummyCalibrateDeviceActivity.java

```diff
@@ -29,6 +29,14 @@ protected void onCreate(Bundle savedInstanceState) {
     public void finishCalibration(View view) {
         // TODO: Not sure if it's best to go back to the device selection or
         // device info page.
+
+        //Added this code for testing purposes with emulator
+        String mac_address = getIntent().getStringExtra(CalibrateDeviceActivity.EXTRAS_DEVICE_ADDRESS);
```

Comment:
Good catch, sorry we didn't update the Dummy activities when we worked on the new calibration.